.hintrc

```diff
@@ -0,0 +1,18 @@
+{
+  "connector": {
+    "name": "local",
+    "options": {
+      "pattern": ["**", "!.git/**", "!node_modules/**"]
+    }
+  },
+  "extends": ["development"],
+  "formatters": ["stylish"],
+  "hints": [
+    "button-type",
+    "disown-opener",
+    "html-checker",
+    "meta-charset-utf-8",
+    "meta-viewport",
+    "no-inline-styles:error"
+  ]
+}
```

Comment:
- [ ] Great on set up the linters files but I think to install the packages for hintrc and stylelint if you scroll down [here](https://github.com/microverseinc/linters-config/tree/master/html-css) you will see the steps please add it to your project ✅
![image](https://user-images.githubusercontent.com/58553711/184869109-2fd349fa-063c-4d4e-a5a1-72ebf482bc4c.png)